### ISSUE TITLE
md-select not displaying icons 

### ISSUE THREAD

**Issue by CosticaPuntaru** [angular/material#3654]
when selecting an option you will expect the select to show the item as is, not the simplified version.
for example if you add flags to the option, when you select an option you expect the select to show the icon even when the dropdown is closed.

how it appears:

![selection_002](https://cloud.githubusercontent.com/assets/6418405/8592247/71faa942-2636-11e5-9c09-9bcf922fb67b.png)

how you expect the select to appear(with the flag):

![material-opend](https://cloud.githubusercontent.com/assets/6418405/8592239/600b77a2-2636-11e5-8bf9-697da2b40232.png)

codepen example:

http://codepen.io/anon/pen/eNrZzO

**Comment by ThomasBurleson** [comment]
@rschmukler - the selected value is currently only a simple string right ?
@CosticaPuntaru - here is an improvement idea for your dropdown rows:

```html
<md-option ng-repeat="state in ctrl.states" value="{{state.abbrev}}">
  <div layout="row" style="width:70px;height:30px;margin-left:10px;padding-top:5px;">
    <img width="15px" height="15px" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACsAAAAwCAMAAAC/knOqAAAAYFBMVEUAAADt7e3n5+eYmJhgYGDX19fi4uITExO/v7/b29smJiZPT0/MzMwcHBx8fHxFRUU2NjaOjo6ysrLPz88MDAy0tLTFxcUuLi6enp5UVFQpKSlmZmZxcXFeXl47OzuGhoas5Ab5AAAA2UlEQVRIie3VXQ+CIBSA4RMGcgBRDmqWH/3/fxnLamQLXVdd+F55tmfTs4kCxBUnrfWkalhL9R01MtTQVedfUK0UQG6b1iOKEHIjuy92IOrh3KI4PGOiXZjnfRznDkY8xHF4e4ope1xkjGVwFAuru6Es5o3LTMqkHU0riZy1RNKjT9qjYGFLbzxHwcKctvOa7DFvsK95t7tNWiTCTdb093NQlw5XrD9H56u0KXtafkAS9iPVxLZLWoCLlTM0roIVe1+0qqp4Ttllu93tr9YxZrfa8I6u/4H/pBvVOQ1Y9SAHgAAAAABJRU5ErkJggg==" />
      <span flex></span>
    {{state.abbrev}}
  </div>
</md-option>
```

**Comment by marcysutton** [comment]
@ThomasBurleson I made one little tweak to your example: including `alt=""` on the image tag means it will be skipped over by a screen reader since there is already text next to it. Otherwise I'm pretty sure the data-URI would be read aloud.

**Comment by CosticaPuntaru** [comment]
the idea is that when the select is closed the flag should be visible

**Comment by GustavoCostaW** [comment]
:+1:

**Comment by GustavoCostaW** [comment]
+1

**Comment by myagoo** [comment]
With the new 0.10.1 release, since this [commit](https://github.com/angular/material/commit/5d9874fb) clearly says:
>BREAKING: `md-select-label` is no longer uses. The select will now
always display the selected value as the text in the `md-option`. This
makes for a better UX for users, as otherwise it was possible to
confuse them"

I was really hoping the md-select directive would extract everything from the md-option and use it as display value.

I know the commit message says "display the selected value as the text in the `md-option`", but it would be an even better UX if the rule was simply : "display the selected value as in the `md-option`"

If this is not implemented, my team and me won't use md-select and switch to md-menu and reimplement everything...

Even if there is style issues, please let us handle it !

**Comment by whoknows** [comment]
+1

**Comment by ThomasBurleson** [comment]
@marcysutton: :+1:

**Comment by nicolas-zozol** [comment]
:+1:

**Comment by myagoo** [comment]
Any news on this ?

**Comment by maelfrize** [comment]
:+1:

**Comment by ptrenery** [comment]
+1

**Comment by ThomasBurleson** [comment]
This will be addressed **after** the v1.0 release.

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our deprecation effort.
    For details, see our post [Spring Cleaning 2016](https://groups.google.com/forum/#!topic/ngMaterial/3Ip9nd68bfk).
